src/commands/ssh/common.rs

```diff
@@ -13,78 +13,47 @@ use crate::controllers::{
     terminal::{SSHConnectParams, TerminalClient},
 };
 
-use super::Args;
-
 /// Creates a connection parameters object from command line arguments
+///
+/// (project, service, environment, deployment instance id)
 pub async fn get_ssh_connect_params(
-    args: Args,
+    args: (
+        Option<String>,
+        Option<String>,
+        Option<String>,
+        Option<String>,
+    ),
```

Comment:
I think we should have a type/struct for these. Otherwise, it is easy to confuse what the correct order should be